src/App.js

```diff
@@ -5,6 +5,7 @@ import { hasLocalToken, setLocalToken } from './lib/token.js';
 import GetList from './Components/getList.js';
 import AddItem from './Components/addItem.js';
 import ShareList from './Components/shareList.js';
+//import FilterList from './Components/listFilter.js'
```

Comment:
Is there any reason you commented this out instead of deleting it?

Reply:
Thank you, I was trying out functionality and rendering the component using dummy data but forgot to remove these when we finished.